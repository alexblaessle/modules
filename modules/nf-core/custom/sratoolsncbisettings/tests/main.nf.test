nextflow_process {

    name "Test Process CUSTOM_SRATOOLSNCBISETTINGS"
    script "modules/nf-core/custom/sratoolsncbisettings/main.nf"
    process "CUSTOM_SRATOOLSNCBISETTINGS"
    tag "custom"
    tag "custom/sratoolsncbisettings"

    test("Should run without failures") {

        when {
            params {
                settings_path = '/tmp/.ncbi'
                settings_file = "${params.settings_path}/user-settings.mkfg"
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
