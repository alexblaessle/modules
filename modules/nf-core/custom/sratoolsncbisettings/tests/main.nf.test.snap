{
    "Should run without failures": {
        "content": [
            [
                "versions.yml:md5,fec13b593c3b42ddd38f2fc77df25b70"
            ]
        ],
        "timestamp": "2023-10-12T10:40:51.717351"
    }
}